env.default

```diff
@@ -28,6 +28,15 @@ AWS_S3_CUSTOM_DOMAIN=
 AWS_SECRET_ACCESS_KEY=
 AWS_STORAGE_BUCKET_NAME=
 
+# Acoustic Config
```

Comment:
Acoustic env variables, we also have DONATION_RECEIPT_METHOD for when we want to make the switch, we can just set it to "DONATE" and it will start sending emails from the donate stack

Reply:
That's a good idea.

Any time we have an env variable that accepts arbitrary strings, let's make sure to document the accepted values so we can more easily figure out what options are available in the future.